src/python/foglamp/translators/omf_translator.py

```diff
@@ -58,19 +30,20 @@
 import aiopg.sa
 import sqlalchemy as sa
 from sqlalchemy.dialects.postgresql import JSONB
-from foglamp import statistics,configuration_manager
+from foglamp import statistics, configuration_manager
+
 # Module information
 __author__ = "${FULL_NAME}"
 __copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
 __license__ = "Apache 2.0"
 __version__ = "${VERSION}"
 
-# FIXME: we need to [SHOULD] move this to defaults.py! unless this also needs to be read from database
+# FIXME: it will be removed using the DB layer
 _DB_URL = 'postgresql:///foglamp'
 """DB references"""
 
-_LOG_SCREEN = True
-"""Enable/Disable screen messages"""
+_TRACE_EXECUTION = False
+"""Enable/Disable info/debug messages"""
 
```

Comment:
Why do we need _TRACE_EXECUTION and method debug_msg_write now if we are using foglamp logger?